README.md

```diff
@@ -282,43 +286,50 @@ coverage report
 ```yml
```

Comment:
This section does not seem relevant -- you should follow the advice given in #100 and fully delete this information.

Reply:
I have deleted this section from the README.